Library/Homebrew/env_config.rb

```diff
@@ -136,6 +136,10 @@ module EnvConfig
                      "use of `curlrc`.",
         boolean:     true,
       },
+      HOMEBREW_CURL_HOME:                        {
+        description: "If set alongside `HOMEBREW_CURLRC`, `curl`(1) will be invoked with `CURL_HOME` " \
```

Comment:
What is `CURL_HOME` used for beyond finding `.curlrc`?

Reply:
I believe that's the only thing -- ideally we could pass a path to `.curlrc` rather than its parent directory, but that doesn't appear possible: https://everything.curl.dev/cmdline/configfile#default-config-file

Reply:
We could perhaps handle that in `HOMEBREW_CURLRC` logic itself so that we set `CURL_HOME` internally to look in the full path specified.

Reply:
Okay, reading the docs _once_ more I see there is actually a `--config` option that could be used!

Is your proposal to change `HOMEBREW_CURLRC` from a boolean to a file path? (In that case I suppose if it's set but _not_ a valid path, we could use the old behavior for compatibility? 😬)

Reply:
I gave that a try, and tested in my environment that needs this. I don't mind that approach at all, actually! Let me know what you think.